components/eamxx/cmake/machine-files/pm-cpu.cmake

```diff
@@ -1,17 +1,8 @@
 include(${CMAKE_CURRENT_LIST_DIR}/common.cmake)
 common_setup()
 
-if ("${PROJECT_NAME}" STREQUAL "E3SM")
-  if (SMP_PRESENT)
-    include (${EKAT_MACH_FILES_PATH}/kokkos/openmp.cmake)
-    #message(STATUS, "pm-cpu openmp SMP_PRESENT=${SMP_PRESENT}")
-  else()
-    include (${EKAT_MACH_FILES_PATH}/kokkos/serial.cmake)
-    #message(STATUS, "pm-cpu serial SMP_PRESENT=${SMP_PRESENT}")
-  endif()
-else()
-  include (${EKAT_MACH_FILES_PATH}/kokkos/openmp.cmake)
-endif()
+include (${EKAT_MACH_FILES_PATH}/kokkos/amd-zen3.cmake)
+include (${EKAT_MACH_FILES_PATH}/kokkos/openmp.cmake)
```

Comment:
@bartgol :

> With the recent changes to the cmake system, turning off the openmp execution space for serial cases is now handled globally in scream/components/cmake/build_eamxx.cmake

Reply:
I still don't think is fine. `build_eamxx.cmake` sets a _normal_ variable, while EKAT sets a _cache_ variable. Since EKAT file is included _last_, it will overwrite the value. I tried this

```
set (FOO "yes")
set (FOO "no" CACHE STRING "")
message ("foo: ${FOO}")
```
and it printed `foo: no`.

Reply:
Ah, wait, we include the eamxx mach file before setting the normal var, so it should be ok.